flintrock/core.py

```diff
@@ -140,8 +140,8 @@ def load_manifest(self, *, user: str, identity_file: str):
             manifest_raw = ssh_check_output(
                 client=master_ssh_client,
                 command="""
-                    cat /home/{u}/.flintrock-manifest.json
-                """.format(u=shlex.quote(user)))
```

Comment:
You're right. I thought it would always quote the username which would make this change ineffective. My mistake. I will fix it.